hadoop-yarn-project/hadoop-yarn/hadoop-yarn-csi/src/test/java/org/apache/hadoop/yarn/csi/client/TestCsiClient.java

```diff
@@ -20,7 +20,7 @@
 
 import csi.v0.Csi;
 import org.apache.commons.io.FileUtils;
-import org.apache.hadoop.test.GenericTestUtils;
+import org.apache.curator.shaded.com.google.common.io.Files;
```

Comment:
Can you use "com.google.common.io.Files" directly instead from curator's jar.?

Reply:
my mistake, thanks you, done